Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/BitmapFileFormat.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// This denotes the format used when saving a bitmap to a file.
+    /// </summary>
+    public enum BitmapFileFormat
+    {
+        /// <summary>
+        /// Indicates Windows Imaging Component's bitmap encoder.
+        /// </summary>
+        Bmp,
+
+        /// <summary>
+        /// Indicates Windows Imaging Component's PNG encoder.
+        /// </summary>
+        Png,
+
+        /// <summary>
+        /// Indicates Windows Imaging Component's bitmap JPEG encoder.
+        /// </summary>
+        Jpeg,
+
+        /// <summary>
+        /// Indicates Windows Imaging Component's TIFF encoder.
+        /// </summary>
+        Tiff,
+
+        /// <summary>
+        /// Indicates Windows Imaging Component's GIF encoder.
+        /// </summary>
+        Gif,
+
+        /// <summary>
+        /// Indicates Windows Imaging Component's JPEGXR encoder.
+        /// </summary>
+        JpegXR
+    }
```

Comment:
Missing Heif for [HeifEncoderId](https://docs.microsoft.com/en-us/uwp/api/windows.graphics.imaging.bitmapencoder.heifencoderid)

Reply:
Thanks, I noticed it, but I found that Heif is not included in [CanvasBitmapFileFormat](http://microsoft.github.io/Win2D/html/T_Microsoft_Graphics_Canvas_CanvasBitmapFileFormat.htm). I need to use the `CanvasBitmapFileFormat` enumeration when saving to other shape images.

Reply:
I've added HEIF encoder support and removed the code for `CanvasBitmapFileFormat`.

HEIF is not supported by Windows by default, so we must install [HEIF Image Extensions](https://www.microsoft.com/en-us/p/heif-image-extensions/9pmmsr1cgpwg) and [HEVC Video Extensions](https://www.microsoft.com/en-us/p/hevc-video-extensions/9nmzlz57r3t7).

Reply:
@michael-hawker On the other hand, I find that `BitmapEncoder` often throws some exceptions that are difficult to solve when using `HeifEncoderId`. Do we really need to support `HeifEncoderId`?

Reply:
Yeah, no, sorry I just noticed it was missing, if it's not stable, please go back.  Thanks!

Reply:
I removed it.